guides/mounts.md

```diff
@@ -32,10 +32,10 @@ Mounts can not contain `/home/container` or any subdirectory of it nor can you c
    - **Target**: The absolute path where the mount will be placed inside of your server, can **not** include `/home/container` in the path.
    - **Read Only**: Whether the mount will be read-only for the servers using it.
    - **User Mountable**: Whether to allow users to self mount this mount.
-4. After creating the mount you are required to add **Eggs** and **Nodes** that this mount may be used on.
+4. After creating the mount, you are required to add **Eggs** and **Nodes** that this mount may be used on.
 
 :::warning Mounts used by multiple servers
-All servers using the same mounts will **only** share its contents when they are on the same node. Mounts are not synchronized between servers.
+All servers using the same mounts will **only** share their contents when they are on the same node. Mounts are not synchronized between servers.
```

Comment:
Change `are not synchronized between servers.` to `are not synchronized between nodes.`